src/main.js

```diff
@@ -0,0 +1,34 @@
+var buttonSection = document.querySelector('.button-section');
+
+buttonSection.addEventListener('click', clickHandler);
+
+function clickHandler(event) {
```

Comment:
This is such an awesome way to reduce the amount of event listeners we have on the page! Love to see it in action!